packages/openstef-models/src/openstef_models/explainability/mixins.py

```diff
@@ -44,6 +45,19 @@ def feature_importances(self) -> pd.DataFrame:
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def predict_contributions(self, data: ForecastInputDataset, *, scale: bool) -> pd.DataFrame:
+        """Get feature contributions for each prediction.
+
+        Args:
+            data: Input dataset for which to compute feature contributions.
+            scale: Whether to scale contributions to sum to the prediction value.
+
+        Returns:
+            DataFrame with contributions per base learner.
+        """
+        raise NotImplementedError
+
```

Comment:
In openstef-meta there is a ContributionsMixin. Should we remove it in favor of this method?

Reply:
Or would it make sense to add ContributionsMixin here and have explainable forecaster extend it. (or have some kind of inbetween protocal approach). So we have a standard interface.